Tools/Frame_params/QuadPlanes/MFE_StriverMini.param

```diff
@@ -6,7 +6,7 @@ SCALING_SPEED    15
 TECS_CLMB_MAX    3
 TECS_LAND_ARSPD  20
 TKOFF_THR_MAX    85
-TRIM_PITCH_CD    100
+PTCH_TRIM_DEG 0.00
```

Comment:
```suggestion
PTCH_TRIM_DEG 1.00
```
If it really should be 0 then we could just remove it from the fie.

Reply:
fixed, it was a bug in my auto-conversion script (fixed now)